templates/repo/user_cards.tmpl

```diff
@@ -19,6 +19,15 @@
 						<span class="octicon octicon-clock"></span> {{$.i18n.Tr "user.join_on"}} {{DateFmtShort .Created}}
 					{{end}}
 				</div>
+
+        {{if $.IsAdmin}}
+          <form class="display inline" action="{{$.RepoLink}}/action/unwatch?user_name={{.Name}}&redirect_to={{$.Link}}" method="POST">
```

Comment:
I think it would be better to use user's ID.

And since this is POST request, use a hidden input `<input type="hidden" name="user_id" value="xxx">` is more ideal.

Reply:
I quite agree.